Dockerfile

```
@@ -0,0 +1,21 @@
+FROM node:10
+
+WORKDIR "/nsd_wrapper"
+
+COPY index.js package.json package-lock.json ./
+
+RUN npm ci
```

If we move `index.js` below `npm ci`, changes in `index.js` (fixes etc) will not trigger rebuild of whole `node_modules`, saves time :)

```suggestion
COPY package.json package-lock.json ./

RUN npm ci

COPY index.js  ./
```